contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -165,7 +164,8 @@ The JWS specification [describes how to encode](https://tools.ietf.org/html/rfc7
 
 #### NEW: `kube-public` namespace
 
-Kubernetes ConfigMaps are per-namespace and are generally only visible to principals that have read access on that namespace.  To create a config map that *everyone* can see, we introduce a new `kube-public` namespace.  This namespace, by convention, is readable by all users (including those not authenticated).
+Kubernetes ConfigMaps are per-namespace and are generally only visible to principals that have read access on that namespace.  To create a config map that *everyone* can see, we introduce a new `kube-public` namespace.  This namespace, by convention, is readable by all users (including those not authenticated). Note that is a convention
```

Comment:
Gah, was indeed thinking about the apiserver-extension configmap in the kube-system namespace. Recalled incorrectly